bigbluebutton-web/src/java/org/bigbluebutton/api/messaging/MessageSender.java

```diff
@@ -79,6 +79,8 @@ private void publish(final String channel, final String message) {
 		  public void run() {
 				Jedis jedis = redisPool.getResource();
 				try {
+					if(channel.equalsIgnoreCase("bigbluebutton:from-bbb-apps:users") || channel.equalsIgnoreCase("bigbluebutton:from-bbb-apps:meeting"))

```

Comment:
Well, this is just for debug. will remove it. BTW, Is there a message flow chart or graph so that there is no need to add those tracing logs.